tools/check_symbol_visibility.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+
+# CI Script to check symbol visibility for nvfuser Python bindings
```

Comment:
👏🏼 Can this be a part of the CI? It is tricky to know when to add `NVF_API`.

Reply:
Also, updating https://github.com/NVIDIA/Fuser/blob/main/doc/dev/visibility.md to include this script would be appreciated.